### ISSUE TITLE
Found your funny Repo

### ISSUE THREAD

**Issue by rafi16jan** [F4uzan/ProtoMaps#2]
Lol man I'm as bored as you but I'm on the other side. I'm searching this Chrome WebAPK "dependency" thing but doesn't found any documentations about it. Even [this](https://developers.google.com/web/fundamentals/integration/webapks) says the APKs can't be published on Play Store, then how Maps Go is published there?

Can you "voluntarily" give me a clue because I don't do Java and not going to crush my mind to find how can I build signed WebAPK with Chrome Dependency or perhaps do you have the Maps Go source? Or can I reverse engine the Maps Go APK to Android Project so I can view the source? Please give me a clue because on the contrary you can remove what I'm searching lol.. What a world we're living

**Comment by rafi16jan** [comment]
Oh yeah, if this work we can make an unofficial documentation to build signed WebAPK that are ready to be published on Play Store. I'm sure there are so many PWA developers that need this

**Comment by F4uzan** [comment]
We can most definitely work our way around from APKs built by Chrome/ium. I haven't thought of this (as I don't use Chrome), but it should be plausible.

This will require backtracking, of course. One that I might or might not do for an evening project. We'll see.

Oh, and as for the 'why can Maps Go be in Play Store?' question — well, let's just say Google is above the law ;)

**Comment by rafi16jan** [comment]
Yeah they're ridiculously above the law. Anyway, are you replacing the Chrome Dependencies (by forking the Maps Go source code and changing it) or creating the WebView from start and linking the URL to Maps Go PWA?

**Comment by F4uzan** [comment]
I did the latter. I was going to do the former, but there is no way I'll read out Chrome's implementation back-to-back for something as silly as this.
